Resources/Locale/en-US/chat/emotes.ftl

```diff
@@ -24,8 +24,8 @@ chat-emote-name-ping = Ping
 chat-emote-name-sneeze = Sneeze
 chat-emote-name-cough = Cough
 chat-emote-name-catmeow = Cat Meow
-chat-emote-name-cathisses = Cat Hisses
-chat-emote-name-monkeyscreeches = Monkey Screeches
+chat-emote-name-cathisses = Cat Hiss
+chat-emote-name-monkeyscreeches = Monkey Screech
```

Comment:
Same as `Crying` above -- these seemed to have been in the third person declarative.